samples/buildKeyAndData.py

```diff
@@ -25,14 +23,12 @@
     cipher = AESHelper(config).create_aescipher_from_config()
 else:
     # Generate new key\iv pair
-    aes_key = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(32))
-    aes_iv = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(16))
-    keys = aes_key + aes_iv
-
-    print 'AES key generated. Key:' + aes_key + " IV:" + aes_iv
+    aes_key = os.urandom(config.aes_key_length)
+    
+    print 'AES key generated. Key:' + aes_key
```

Comment:
Are the print statements going to go to STDOUT so they get picked up by BAC logging?

Reply:
This is internal script, not used on VMs